.github/workflows/ci.yaml

```diff
@@ -0,0 +1,56 @@
+name: "CI"
+
+on:
+  pull_request:
+  push:
+    branches:
+      - "master"
+  schedule:
+    - cron: "42 3 * * *"
+
+jobs:
+  phpunit:
+    name: "PHPUnit"
+    runs-on: "ubuntu-20.04"
+
+    strategy:
+      matrix:
+        php-version:
+          - "5.6"
+          - "7.0"
+          - "7.1"
+          - "7.2"
+          - "7.3"
+          - "7.4"
+        dependencies:
+          - "highest"
+        include:
+          - dependencies: "lowest"
+            php-version: "5.6"
+
+    steps:
+      - name: "Checkout"
+        uses: "actions/checkout@v2"
+        with:
+          fetch-depth: 2
+
+      - name: "Install PHP"
+        uses: "shivammathur/setup-php@v2"
+        with:
+          php-version: "${{ matrix.php-version }}"
+          coverage: "pcov"
+          ini-values: "zend.assertions=1"
+
+      - name: "Install dependencies with Composer"
+        uses: "ramsey/composer-install@v1"
+        with:
+          dependency-versions: "${{ matrix.dependencies }}"
+
+      - name: "Run PHPUnit"
+        run: "vendor/bin/phpunit --coverage-clover=coverage.xml"
+
+      - name: "Upload coverage file"
+        uses: "actions/upload-artifact@v2"
+        with:
+          name: "phpunit-${{ matrix.deps }}-${{ matrix.php-version }}.coverage"
+          path: "coverage.xml"
```

Comment:
I don't think this task is needed.

Problems Scrutinizer CI and Coveralls jobs should be added here.

```yaml
      - name: Send coverage results to Scrutinizer CI
        run: |
          wget https://scrutinizer-ci.com/ocular.phar
          php ocular.phar code-coverage:upload --format=php-clover coverage.xml

      - name: Send coverage results to Coveralls
        env:
          COVERALLS_REPO_TOKEN: ${{ secrets.GITHUB_TOKEN }}
          COVERALLS_PARALLEL: true
          COVERALLS_FLAG_NAME: 'PHP ${{ matrix.php }}
        run: |
          composer global require php-coveralls/php-coveralls
          php-coveralls -x build/coverage-clover.xml -o build/coveralls-upload.json -v
```

Reply:
Do you insist on using these two (scrutinizer and coveralls. E.g. it took a year to coveralls since GA was supported.)?

I can setup Codecov which is up to date and even has its own action.

Reply:
Good. Let's move from Coveralls to Codecov :+1: